workflows/scrnaseq.nf

```diff
@@ -131,6 +140,27 @@ workflow SCRNASEQ {
         ch_multiqc_fastqc = Channel.empty()
     }
 
+    //
+    // Uncompress genome fasta file if required
+    //
+    if (params.fasta.endsWith('.gz')) {
+        ch_genome_fasta    = GUNZIP_FASTA ( [ [:], file(params.fasta) ] ).gunzip.map { it[1] }
+        ch_versions        = ch_versions.mix(GUNZIP_FASTA.out.versions)
+    } else {
+        ch_genome_fasta = Channel.value( file(params.fasta) )
+    }
+
+    //
+    // Uncompress GTF annotation file or create from GFF3 if required
+    //
+    if (params.gtf.endsWith('.gz')) {
+        ch_gtf      = GUNZIP_GTF ( [ [:], file(params.gtf) ] ).gunzip.map { it[1] }
+        ch_versions = ch_versions.mix(GUNZIP_GTF.out.versions)
+    } else {
+        ch_gtf = Channel.value( file(params.gtf) )
+    }
```

Comment:
It should stil be possible to run the workflow when *just* a cellranger index, but no fasta file is provided. 

This was previously possible due to `if (!params.fasta) { ch_genome_fasta = [] }`, but it fails now.

Reply:
Applied the changes so that it is possible again to run without `params.fasta` and/or `params.gtf`.